custom_components/tuya_local/devices/beca_zht002_thermostat_c.yaml

```diff
@@ -0,0 +1,93 @@
+name: Zigbee Thermostat
```

Comment:
Leave out Zigbee from the top level name, as they may make WiFi and Bluetooth versions that match the same config.